### PR TITLE
RBA Script - Add Teams Warning

### PR DESCRIPTION
**Pull request by Shanefe**
**Issue:**
Issue came to PG where teams info was deleted by RBA. 

**Reason:**
This was by design as they set the RBA to delete the body

**Fix:**
Explicit warning that this will removed Teams info

**Validation:**
Provide if applicable

### CODE REVIEW

**Comment by Shanefe** [comment]
CodeFormatter.ps1 does not seems to be working for me anymore.
Any suggestions?

.\.build\CodeFormatter.ps1 -save -branch main
Optimize Code: True
Get-ChildItem: D:\src\CSS-Exchange\.build\CodeFormatter.ps1:48
Line |
  48 |  … k = $filesFullPath | Get-ChildItem -Include "*.ps1", "*.psm1", "*.md"
     |                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot find path 'D:\src\CSS-Exchange\.build\HelpFunctions\Calendar' because it does not exist.
**No scripts or md files were modified. Skipping over check.**

**Comment by dpaulson45** [comment]
> CodeFormatter.ps1 does not seems to be working for me anymore. Any suggestions?
> 
> ..build\CodeFormatter.ps1 -save -branch main Optimize Code: True Get-ChildItem: D:\src\CSS-Exchange.build\CodeFormatter.ps1:48 Line | 48 | … k = $filesFullPath | Get-ChildItem -Include "_.ps1", "_.psm1", "*.md" | ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ | Cannot find path 'D:\src\CSS-Exchange.build\HelpFunctions\Calendar' because it does not exist. **No scripts or md files were modified. Skipping over check.**

I have no idea how you are running it. but it saying that you don't have a path because it doesn't exist. We do not have a directory called `CSS-Exchange.build`.

File: Calendar/Get-RBASummary.ps1 [hunk 1]
```diff
@@ -310,6 +310,7 @@ function RBADelegateSettings {
     Write-Host "`t ForwardRequestsToDelegates:      "$RbaSettings.ForwardRequestsToDelegates
     Write-Host
 
+    # Warnings:
```
Comment (dpaulson45) [inline Calendar/Get-RBASummary.ps1:313]: You call out a warning but then don't do a `Write-Host` with `-ForegroundColor Yellow`

Comment (Shanefe) [reply Calendar/Get-RBASummary.ps1]: That is just the block header for the next 30 lines of code.  They are all checking for issues.

Updated comments to make it more clear.

**Review by dpaulson45 — APPROVED** [review]

**Pull request merged**